### DISCUSSION TITLE
Default poster colour looks bad on non-white backgrounds

### DISCUSSION THREAD

**Discussion by max-morawski-photo** [google/model-viewer#3459] [Q&A]
### Description

When the modelviewer is not displaying a model yet the poster is displayed, defaulting to solid white colour. This looks bad on pages with non-white background.

### Version

Latest from CDN.

### Browser Affected
All

### OS
All

### Workaround

The button style currently has this as the default background colour:
`background-color: var(--poster-color, #fff)`

Took me a bit of time to find the offending element, but the below fixes the issue:
```
<style>
  :root { 
    --poster-color: rgba(255, 255, 255, 0);
  }
</style>
```

**Comment by samaneh-kazemi** [comment]
Yes you can update the poster background color to be transparent.

https://modelviewer.dev/docs/#loading-css

**Reply by max-morawski-photo** [reply]
That is my workaround indeed 👆🏽. As a regular user, I had no idea I was looking for something called a _poster_ though 😉 So I ended up hunting down the DOM tree in the web debugger until I found the offending element.

What I am suggesting is to change the default background to transparency. If someone is working with posters they would know that they exist and what they do. With the current colour, however, an unsuspecting user is left none the wiser as to why their viewer has a weird white background.

I should have made it clearer perhaps, this is not a technical bug report (well done for making it a CSS variable 👍🏽) but a user experience issue report.

**Reply by elalish** [reply]
I agree, and it only hasn't happened yet because it would be a breaking change. Please take a look at our v2.0 [proposal](https://github.com/google/model-viewer/discussions/3460), which includes this, where you can voice your support.
